### DISCUSSION TITLE
Configurations for disabling @error highlighting

### DISCUSSION THREAD

**Discussion by zhangzhang10** [nvim-treesitter/nvim-treesitter#5471] [General]
The default behavior of highlighting syntax errors while typing code is undesirable at best. I know I can type a command to disable it for the current buffer:
`:highlight link @error Normal`

But how do I modify `nvim-treesitter.configs` to disable it by default? 

Besides, how do I selectively disable it for certain languages, for example, only C and C++ but keep it on for other languages?

Thanks.

**Comment by amaanq** [comment]
Your theme should just not be setting it to begin with, file it w/ them

**Comment by zhangzhang10** [comment]
Thanks for the hint. Resolved.
